### ISSUE TITLE
White Line in Morse Varient

### ISSUE THREAD

**Issue by mktufan21** [kolking/react-native-page-indicator#12]
Hi @kolking  , when we are using morse there is a white-line , please check the image, any idea how we can fix it

`<PageIndicator
        count={images.length}
        current={currentPage}
        color={'#D3DAE1'}
        activeColor={'#606C70'}
        size={moderateScale(8,0.3)}
        variant='morse'
        dashSize={moderateScale(8,0.3)*4}
        borderRadius={moderateScale(8,0.3)/2}
      />`

<img width="339" alt="Screenshot 2023-08-16 at 7 14 55 PM" src="https://github.com/kolking/react-native-page-indicator/assets/109002345/3a9e138f-a9b2-46f4-acbf-29057b3c1c39">

**Comment by kolking** [comment]
I suppose this is caused by some non-integer values produced by the `moderateScale()` function. Try something like this:

```tsx
const size = Math.round(moderateScale(8,0.3));

<PageIndicator count={images.length} current={currentPage} color={'#D3DAE1'} activeColor={'#606C70'} size={size} variant='morse' dashSize={size*4} borderRadius={size/2} />
```

Btw, you don’t have to explicitly define `dashSize={size*4}` and `borderRadius={size/2}` since these are default. Thus, the below will produce the same result as above:

```tsx
const size = Math.round(moderateScale(8,0.3));

<PageIndicator count={images.length} current={currentPage} color={'#D3DAE1'} activeColor={'#606C70'} size={size} variant='morse' />
```

**Comment by chrissikora** [comment]
Actually I am experiencing similar behaviour - very thin white line and my values are integers:

<img width="90" alt="Screenshot 2023-08-23 at 9 22 16 am" src="https://github.com/kolking/react-native-page-indicator/assets/69775435/2d33ae81-61be-40b6-9fc5-1bbed73ae794">

Values:
```
size={8}
dashSize={18}
```

**Comment by kolking** [comment]
@chrissikora thanks for letting me know! I will do my best to figure out and fix the issue asap.

**Comment by kolking** [comment]
@chrissikora could you please provide more information since I wasn't able to reproduce the issue with the `size` and `dashSize` values you've mentioned. Are you seeing this in iOS or Android? Simulator or real device? What are the full list of props you're using for the component?

**Comment by mktufan21** [comment]
@kolking little bit late , but this rounded does not fix the issue . for your above comment the issue I am facing in android emulator

**Comment by kolking** [comment]
@mktufan21 @chrissikora the issue should be resolved in `v2.2.0`. Appreciate your feedback, thanks!
